doc/man7/ossl-guide-introduction.pod

```diff
@@ -0,0 +1,92 @@
+can obtain that source code via the OpenSSL website
+(L<https://www.openssl.org/source>).
+
+Many Operating Systems (notably Linux distributions) supply pre-built OpenSSL
+binaries either pre-installed or available via the package management system in
+use for that OS. It is worth checking whether this applies to you before
+attempting to build OpenSSL from the source code.
+
+Some third parties also supply OpenSSL binaries (e.g. for Windows and some other
+platforms). The OpenSSL project maintains a list of these third parties at
+L<https://wiki.openssl.org/index.php/Binaries>.
+
+If you build and install OpenSSL from the source code then you should download
+the appropriate files for the version that you want to use from the link given
+above. Extract the contents of B<tar.gz> archive file that you downloaded into
+an appropriate directory. Inside that archive you will find a file named
+B<INSTALL.md> which will supply detailed instructions on how to build and
+install OpenSSL from source. Make sure you read the contents of that file
+carefully in order to achieve a successful build. In the directory you will also
+find a set of B<NOTES> files that provide further platform specific information.
+Make sure you carefully read the file appropriate to your platform. As well as
+the platform specific B<NOTES> files there is also a B<NOTES-PERL.md> file that
+provides information about setting up Perl for use by the OpenSSL build system
+across multiple platforms.
+
+Sometimes you may want to build and install OpenSSL from source on a system
+which already has a pre-built version of OpenSSL installed on it via the
+Operating System package management system (for example if you want to use a
+newer version of OpenSSL than the one supplied by your Operating System). In
+this case it is recommended to install OpenSSL to a different location than
+where the pre-built version is installed. You should never replace the pre-built
+version with a different version as this may break your system.
+
+=head1 CONTENTS OF THE OPENSSL GUIDE
+
+The OpenSSL Guide is a series of documentation pages (starting with this one)
+that introduce some of the main concepts in OpenSSL. The guide can either be
+read end-to-end in order, or alternatively you can simply skip to the parts most
+applicable to your use case. Note however that later pages may depend on and
+assume knowledge from earlier pages.
+
+The pages in the guide are as follows:
+
+L<ossl-guide-libraries-introduction(7)>: An introduction to the OpenSSL libraries
+
+L<ossl-guide-libcrypto-introduction(7)>: An introduction to libcrypto
+
+L<ossl-guide-libssl-introduction(7)>: An introduction to libssl
+
+L<ossl-guide-tls-introduction(7)>: An introduction to SSL/TLS in OpenSSL
+
+L<ossl-guide-tls-client-block(7)>: Writing a simple blocking TLS client
+
+L<ossl-guide-quic-introduction(7)>: An introduction to QUIC in OpenSSL
+
+L<ossl-guide-quic-client-block(7)>: Writing a simple blocking QUIC client
```

Comment:
Ought this be a bulleted or numbered list where the text links to the page?  I.e.

- [An introduction to the OpenSSL libraries](ossl-guide-libraries-introduction.htm)
- [An introduction to libcrypto](ossl-guide-libcrypto-introduction.htm)
- [An introduction to libssl](ossl-guide-libssl-introduction.htm)
- [An introduction to SSL/TLS in OpenSSL](ossl-guide-tls-introduction.htm)
- [Writing a simple blocking TLS client](ossl-guide-tls-client-block.htm)
- [An introduction to QUIC in OpenSSL](ossl-guide-quic-introduction.htm)
- [Writing a simple blocking QUIC client](ossl-guide-quic-client-block.htm)